platform_main_feature.strongarm.inc

```diff
@@ -219,6 +219,16 @@ function platform_main_feature_strongarm() {
     'islandora:root' => 'islandora:root',
     'islandora:transformCollection' => 'islandora:transformCollection',
     'islandora:video_collection' => 'islandora:video_collection',
+    'islandora:5' => 0,
```

Comment:
Need to take a look into this.

Reply:
Silly me. These are collection search pid's, and can be removed for now